index.Rmd

```diff
@@ -153,126 +157,121 @@ the assignment symbol is preferred and is what we will use in this course. When
-The second argument is a logical statement to use when filtering the rows. Here, for example, we'll say that we are interested in rows where the state is New York. To make 
-this comparison, we use the *equivalency operator* `==` to compare the values of the `state` column with the value `"New York"`. Similar to when we loaded the data file and put quotes around the filename, 
-here we need to put quotes around `"New York"` to tell R that this is a character value and not one of the special words that make up R programming language, nor one of the names 
+We can use the `filter` function to obtain the subset of rows with desired values from a data frame. Again, our first argument is the name of the data frame object, `can_lang`. 
+The second argument is a logical statement to use when filtering the rows. Here, for example, we'll say that we are interested in rows where the language is Mandarin. To make 
+this comparison, we use the *equivalency operator* `==` to compare the values of the `language` column with the value `"Mandarin"`. Similar to when we loaded the data file and put quotes around the filename, 
+here we need to put quotes around `"Mandarin"` to tell R that this is a character value and not one of the special words that make up R programming language, nor one of the names 
 we have given to data frames in the code we have already written.
 
 With these arguments, `filter` returns a data frame that has all the columns of the input data frame but only the rows we asked for in our logical filter statement. 
 
 ```{r}
-new_york <- filter(us_data, state == "New York")
-new_york
+mandarin <- filter(can_lang, language == "Mandarin")
+mandarin
 ```
 
 ### Using `filter` to extract rows with values above a threshold
 
 
-If we are interested in finding information about the states who have a higher median household income
- than New York---whose median household income is \$58,771---then we can create a filter 
-to obtain rows where the value of `med_income` is greater than 58771.
+If we are interested in finding information about the languages who have a higher number of people who primarily speak it at home compared to Mandarin---which is reported to have 462890 people speaking it as the primary language they speak in their home---then we can create a filter 
+to obtain rows where the value of `most_at_home` is greater than 462890.
 In this case, we see that `filter` returns a data frame with 16 rows; this indicates that there are 16 
 states with higher median household incomes than New York.
 
 ```{r}
-high_incomes <- filter(us_data, med_income > 58771)
-high_incomes
+spoke_often_at_home <- filter(can_lang, most_at_home > 462890)
+spoke_often_at_home
 ```
 
 
 ## Exploring data with visualizations
 
 Creating effective data visualizations is an essential piece to any data analysis. For the remainder of Chapter 1, we will learn how to use
```

Comment:
Suggestion for this paragraph: 

Creating effective data visualizations is an essential piece of any data analysis. For the remainder of Chapter 1, we will learn how to use functions from the tidyverse package to make visualizations that let us explore relationships in data. In particular, we’ll develop a visualization of the language data collected in the 2016 Canadian census we’ve been working with that will help us understand two potential relationships in the data. First, we will explore the relationship between the number of people who speak a language as their mother tongue and the number of people who speak it as their primary language at home. And second, we will explore whether there a pattern in the strength of this relationship in the higher-level language categories (Official languages, Aboriginal languages, or non-official and non-Aboriginal languages). These questions are examples of exploratory data analysis questions: we are looking for relationships and patterns within the data set we have, but are not trying to generalize what we find to others beyond this data set.